waspc/data/Generator/templates/sdk/auth/useAuth.ts

```diff
@@ -2,7 +2,7 @@
 import { deserialize as superjsonDeserialize } from 'superjson'
 import { useQuery, addMetadataToQuery } from 'wasp/client/operations'
 import { api, handleApiError } from 'wasp/client/api'
-import { HttpMethod } from 'wasp/types'
+import { HttpMethod } from 'wasp/client'
```

Comment:
Maybe this is even general enough to export from `wasp`? Probably too much though.

Reply:
Was thinking the same, but probably it doesn't matter much now.

Reply:
It has been used on the client so for, so I felt this made sense. But I get the general feel that `wasp` would make sense.